pkg/asset/manifests/network.go

```diff
@@ -129,6 +136,41 @@ func (no *Networking) Generate(dependencies asset.Parents) error {
                 },
         }
 
+	// Untar and add acc-provision files
+	r, _ := os.Open(installConfig.Config.Platform.OpenStack.AciNetExt.ProvisionTar)
+	uncompressedStream, _ := gzip.NewReader(r)
+	tarReader := tar.NewReader(uncompressedStream)
+	for true {
+		header, err := tarReader.Next()
+		if err == io.EOF {
+			break
+		}
+		b, _ := ioutil.ReadAll(tarReader);
```

Comment:
I have un-tarred the same file before in the validation phase, and error was handled there. So didn't think we need it again, since the code won't reach this point if there was a problem

Reply:
ok thanks